visual-dotnet/SauceLabs.Visual/VisualCheckOptions.cs

```diff
@@ -30,7 +30,7 @@ public class VisualCheckOptions
 
         internal void EnsureTestContextIsPopulated(string callerMemberName, string? previousSuiteName)
         {
-            if (string.IsNullOrEmpty(callerMemberName) || HasIncompleteTestContext())
+            if (string.IsNullOrEmpty(callerMemberName) || !HasIncompleteTestContext())
```

Comment:
is this correct?

Reply:
Yes, there is an actual bug in the main branch.  I can make another PR for that if needed.